fabric-data-generation-api-v1/src/testmod/java/net/fabricmc/fabric/test/datagen/DataGeneratorTestEntrypoint.java

```diff
@@ -114,6 +119,14 @@ public void onInitializeDataGenerator(FabricDataGenerator dataGenerator) {
 		}
 	}
 
+	@Override
+	public void buildRegistry(RegistryBuilder registryBuilder) {
+		registryBuilder.addRegistry(
+				DataGeneratorTestContent.TEST_DATAGEN_DYNAMIC_REGISTRY_KEY,
+				(registerable) -> { }
```

Comment:
I would create a full method for this and register the test entry to this `Registerable` see TestBiomes.bootstrap()